### ISSUE TITLE
State wrapper doesn't create cards for all kicked soccer balls

### ISSUE THREAD

**Issue by matthew-blackman** [phetsims/center-and-variability#336]
When working on #262, the devs found that when loading the state wrapper, not all kicked soccer balls have a card. This is most likely a problem in the state wrapper, but we discovered it using Studio -> Test

![Screen Shot 2023-06-28 at 1 33 58 PM](https://github.com/phetsims/center-and-variability/assets/108756567/523dca0a-19c4-4aa8-8a05-c207939a41f7)

**Comment by samreid** [comment]
Likely we would want to implement #314 before working on this.  I'll add a hold for now.

**Comment by marlitas** [comment]
I think this was addressed by work done in: https://github.com/phetsims/center-and-variability/issues/347

Including in that review.

**Comment by matthew-blackman** [comment]
Tested cards fairly extensively in the state wrapper and it's is looking awesome! All cards are being created and cell positions are being updated correctly both with and without 'Sort Data' checked. Nice work @marlitas!

I noticed two small discrepancies in the state wrapper (see screenshot below) that I don't think are issues but wanted to note in case we do future work here:
1. Picking up a card (small translation up and to the left) does not cause the corresponding downstream card to get picked up. I don't think this is an issue since it is a input-linked behavior that shouldn't be stateful anyway, but wanted to note.
2. Sometimes the z-order of the cards in the downstream sim doesn't match the upstream sim, but this is only if the cards are in an intermediate position while being sorted or dragged. So I also think this is fine since the cards' cell position is really all that needs to be stateful here.

<img width="509" alt="CaV_state_wrapper" src="https://github.com/phetsims/center-and-variability/assets/108756567/80b065b6-ffab-4c9d-9372-6b02a7e35f22">

@marlitas feel free to open issues related to these small discrepancies if you see fit, but I think that users/designers will have everything they need with the current statefulness and behavior of cards. Closing this issue.
